profiles/desktop.py

```diff
@@ -52,7 +52,7 @@ def _prep_function(*args, **kwargs) -> bool:
 	other code in this stage. So it's a safe way to ask the user
 	for more input before any other installer steps start.
 	"""
-	choice = Menu(str(_('Select your desired desktop environment')), __supported__).run()
+	choice = Menu(str(_('Select your desired desktop environment')), __supported__, skip=True).run()
```

Comment:
[Skip is True by default](https://github.com/archlinux/archinstall/blob/master/archinstall/lib/menu/menu.py#L48)

Reply:
Yeah **True** is redundant